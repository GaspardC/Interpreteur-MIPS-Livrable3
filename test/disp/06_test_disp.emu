#-------
#TEST_RETURN_CODE=PASS           
#TEST_COMMENT="Teste fonctionnement standard"
#-------

disp reg $3 $32 $8 $9
