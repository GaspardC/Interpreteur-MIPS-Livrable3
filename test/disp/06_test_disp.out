v1 : 0 PC : 0 t0 : 0 t1 : 0 
